src/FakeItEasy.Analyzer/UnusedReturnValueAnalyzer.cs

```diff
@@ -64,19 +63,15 @@ private static void AnalyzeCall(SyntaxNodeAnalysisContext context)
         private static IMethodSymbol GetCalledMethodSymbol(InvocationExpressionSyntax call, SyntaxNodeAnalysisContext context)
         {
             var memberAccess = call?.Expression as MemberAccessExpressionSyntax;
-            var methodSymbol = context.SemanticModel.GetSymbolInfo(memberAccess?.Name).Symbol as IMethodSymbol;
-
-            return methodSymbol;
+            return context.SemanticModel.GetSymbolInfo(memberAccess?.Name).Symbol as IMethodSymbol;
         }
 
         private static string GetDiagnosticName(IMethodSymbol methodSymbol)
         {
-            var attribute =
-                methodSymbol.GetAttributes()
-                    .FirstOrDefault(a => a.AttributeClass.GetFullName() == "FakeItEasy.Analysis.MustUseReturnValueAttribute");
+            var attribute = methodSymbol.GetAttributes()
+                .FirstOrDefault(a => a.AttributeClass.GetFullName() == "FakeItEasy.Analysis.MustUseReturnValueAttribute");
 
-            var argument = attribute?.ConstructorArguments.FirstOrDefault();
-            return argument?.Value as string;
+            return attribute?.ConstructorArguments.FirstOrDefault().Value as string;

```

Comment:
I wanted to collapse this down to one statement but StyleCop blows up if you put a line break before `?.`.

> Severity	Code	Description	Project	File	Line	Suppression State
Warning		SA1003 : CSharp.Spacing : The spacing around the symbol '?.' is invalid.	FakeItEasy.Analyzer	C:\code\FakeItEasy\FakeItEasy\src\FakeItEasy.Analyzer\UnusedReturnValueAnalyzer.cs	75	

I've raised https://stylecop.codeplex.com/workitem/7722